demo/java-swing/README.md

```diff
@@ -4,6 +4,19 @@ A voice-controlled smart lighting panel using Java Swing and the Picovoice platf
 
 Before running the demo, run the `copy.sh` script to copy wake word and context files from elsewhere in the repo.
 
+Build the app with Gradle:
+```console
+cd porcupine/demo/java-swing
+./gradlew build
+```
+
+Then launch the app:
+
+```console
+cd porcupine/demo/java-swing/build/libs
+java -jar JavaSwingDemo.jar
```

Comment:
Everything executed fine for me, but when I ran `copy.sh` it looks like it's trying to copy the old binding Jar file. Just deleting that command should work.

```console
$ sh copy.sh 
Copying Linux context...
Copying Mac context...
Copying Windows context...
Copying Linux keyword file...
Copying Mac keyword file...
Copying Windows keyword file...
Copying Picovoice JAR
cp: ../../sdk/java/bin/*.jar: No such file or directory
```